docs/proposals/deploy-csi-to-k3s.md

```diff
@@ -0,0 +1,54 @@
+# Proposal: Deploy CSI on K3s cluster 
+
+Last updated: 8.04.22
+
+
+## Abstract
+
+Deploy CSI on K3s cluster, test it work and add steps for manual patching scheduler 
+
+## Background
+
+Currently csi is compatible with Vanilla Kubernetes, RKE2, OpenShift and Kind, but only for test purpose. We want to deploy csi to new platform.  
```

Comment:
It sounds like all platforms are for test purpose. Please rephrase your sentence.
We want to support new platform, not deploy.